Data/market-hours/market-hours-database.csv

```diff
@@ -10,4 +10,5 @@
 #
 America/New_York,usa,,equity,-,-,-,-,4,9.5,16,20,4,9.5,16,20,4,9.5,16,20,4,9.5,16,20,4,9.5,16,20,-,-,-,-
 # fxcm data uses EST, does not use EDT, so use a fixed offset
-UTC-05,fxcm,,forex,17,17,24,24,+,+,+,+,+,+,+,+,+,+,+,+,+,+,+,+,0,0,17,17,-,-,-,-
\ No newline at end of file
+UTC-05,fxcm,,forex,17,17,24,24,+,+,+,+,+,+,+,+,+,+,+,+,+,+,+,+,0,0,17,17,-,-,-,-
+America/Sao_Paulo,bra,,equity,-,-,-,-,9.5,10,16.9166667,18,9.5,10,16.9166667,18,9.5,10,16.9166667,18,9.5,10,16.9166667,18,9.5,10,16.9166667,18,-,-,-,-
```

Comment:
Hey Alex, I think this should be moved to another pull-request. Do the SP market hours really close at 16.55?